lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/parameters/AzureBatchAccountUri.java

```diff
@@ -16,12 +16,14 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 package org.apache.reef.runime.azbatch.parameters;
 
 import org.apache.reef.tang.annotations.NamedParameter;
 import org.apache.reef.tang.annotations.Name;
 
+/**
+ * The Azure Batch account URI.
+ */
 @NamedParameter(doc = "The Azure Batch Account URL")
```

Comment:
URL -> URI. 

I have this in my PR tho. You could ignore it if you want.

Reply:
fixed.